web_generator/lib/src/_js_supertypes_src.dart

```diff
@@ -0,0 +1,4 @@
+import 'dart:js_interop';
```

Comment:
It took me a while to figure out the purpose of this file. :D I wonder if we should instead create a temporary file for analysis that we delete after calculating the JS type supertypes. The contents could just be what we have here.